<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1029</width>
    <height>653</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Cold Atom Capture</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QWidget" name="widget" native="true">
      <layout class="QHBoxLayout" name="horizontalLayout_3">
       <item>
        <widget class="QTabWidget" name="tabWidget">
         <property name="currentIndex">
          <number>0</number>
         </property>
         <widget class="QWidget" name="cameraTab">
          <attribute name="title">
           <string>Camera</string>
          </attribute>
          <layout class="QHBoxLayout" name="horizontalLayout_2">
           <item>
            <layout class="QVBoxLayout" name="settingsLayout">
             <item>
              <widget class="QGroupBox" name="camSettingsBox">
               <property name="title">
                <string>Settings</string>
               </property>
               <layout class="QVBoxLayout" name="verticalLayout_6">
                <item>
                 <layout class="QHBoxLayout" name="tofStartLayout">
                  <item>
                   <widget class="QLabel" name="tofLabel">
                    <property name="text">
                     <string>TOF start (ms):</string>
                    </property>
                   </widget>
                  </item>
                  <item>
                   <widget class="QDoubleSpinBox" name="tofStartBox">
                    <property name="minimumSize">
                     <size>
                      <width>100</width>
                      <height>0</height>
                     </size>
                    </property>
                    <property name="maximumSize">
                     <size>
                      <width>100</width>
                      <height>16777215</height>
                     </size>
                    </property>
                   </widget>
                  </item>
                 </layout>
                </item>
                <item>
                 <layout class="QHBoxLayout" name="tofEndLayout">
                  <item>
                   <widget class="QLabel" name="label">
                    <property name="text">
                     <string>TOF end (ms):</string>
                    </property>
                   </widget>
                  </item>
                  <item>
                   <widget class="QDoubleSpinBox" name="tofEndBox">
                    <property name="minimumSize">
                     <size>
                      <width>100</width>
                      <height>0</height>
                     </size>
                    </property>
                    <property name="maximumSize">
                     <size>
                      <width>100</width>
                      <height>16777215</height>
                     </size>
                    </property>
                   </widget>
                  </item>
                 </layout>
                </item>
                <item>
                 <layout class="QHBoxLayout" name="tofStepsLayout">
                  <item>
                   <widget class="QLabel" name="tofSplitLabel">
                    <property name="text">
                     <string>TOF Steps:</string>
                    </property>
                   </widget>
                  </item>
                  <item>
                   <widget class="QSpinBox" name="tofSplitBox">
                    <property name="minimumSize">
                     <size>
                      <width>100</width>
                      <height>0</height>
                     </size>
                    </property>
                    <property name="maximumSize">
                     <size>
                      <width>100</width>
                      <height>16777215</height>
                     </size>
                    </property>
                   </widget>
                  </item>
                 </layout>
                </item>
                <item>
                 <layout class="QHBoxLayout" name="exposureLayout">
                  <item>
                   <widget class="QLabel" name="exposureLabel">
                    <property name="maximumSize">
                     <size>
                      <width>16777215</width>
                      <height>30</height>
                     </size>
                    </property>
                    <property name="text">
                     <string>Exposure Time (Î¼s)</string>
                    </property>
                   </widget>
                  </item>
                  <item>
                   <widget class="QSpinBox" name="exposureBox">
                    <property name="minimumSize">
                     <size>
                      <width>100</width>
                      <height>0</height>
                     </size>
                    </property>
                    <property name="maximumSize">
                     <size>
                      <width>100</width>
                      <height>16777215</height>
                     </size>
                    </property>
                    <property name="minimum">
                     <number>100</number>
                    </property>
                    <property name="maximum">
                     <number>30000002</number>
                    </property>
                   </widget>
                  </item>
                 </layout>
                </item>
                <item>
                 <layout class="QHBoxLayout" name="roiSelectionLayout">
                  <item>
                   <widget class="QLabel" name="label_2">
                    <property name="text">
                     <string>ROI Generation:</string>
                    </property>
                   </widget>
                  </item>
                  <item>
                   <widget class="QRadioButton" name="roiAutoRadio">
                    <property name="text">
                     <string>Automatic</string>
                    </property>
                    <property name="checked">
                     <bool>true</bool>
                    </property>
                   </widget>
                  </item>
                  <item>
                   <widget class="QRadioButton" name="roiManualRadio">
                    <property name="text">
                     <string>Manual</string>
                    </property>
                   </widget>
                  </item>
                 </layout>
                </item>
                <item>
                 <layout class="QHBoxLayout" name="sigmaLayout">
                  <item>
                   <widget class="QLabel" name="sigmaLabel">
                    <property name="text">
                     <string>Sigma Factor:</string>
                    </property>
                   </widget>
                  </item>
                  <item>
                   <widget class="QSpinBox" name="sigmaBox">
                    <property name="minimumSize">
                     <size>
                      <width>100</width>
                      <height>0</height>
                     </size>
                    </property>
                    <property name="maximumSize">
                     <size>
                      <width>100</width>
                      <height>16777215</height>
                     </size>
                    </property>
                    <property name="minimum">
                     <number>1</number>
                    </property>
                   </widget>
                  </item>
                 </layout>
                </item>
                <item>
                 <layout class="QHBoxLayout" name="roiManualLayout">
                  <item>
                   <widget class="QLabel" name="roiManualLabel">
                    <property name="maximumSize">
                     <size>
                      <width>16777215</width>
                      <height>75</height>
                     </size>
                    </property>
                    <property name="text">
                     <string>ROI Corners:</string>
                    </property>
                   </widget>
                  </item>
                  <item>
                   <widget class="QLineEdit" name="roiTopLeftEdit">
                    <property name="enabled">
                     <bool>false</bool>
                    </property>
                    <property name="maximumSize">
                     <size>
                      <width>100</width>
                      <height>16777215</height>
                     </size>
                    </property>
                    <property name="placeholderText">
                     <string>x1,y1</string>
                    </property>
                   </widget>
                  </item>
                  <item>
                   <widget class="QLineEdit" name="roiBottomRightEdit">
                    <property name="enabled">
                     <bool>false</bool>
                    </property>
                    <property name="maximumSize">
                     <size>
                      <width>100</width>
                      <height>16777215</height>
                     </size>
                    </property>
                    <property name="inputMask">
                     <string/>
                    </property>
                    <property name="text">
                     <string/>
                    </property>
                    <property name="placeholderText">
                     <string>x2,y2</string>
                    </property>
                   </widget>
                  </item>
                 </layout>
                </item>
                <item>
                 <layout class="QHBoxLayout" name="recallLayout">
                  <item>
                   <widget class="QLabel" name="recallDateLabel">
                    <property name="text">
                     <string>Date:</string>
                    </property>
                   </widget>
                  </item>
                  <item>
                   <widget class="QDateEdit" name="recallDateBox">
                    <property name="enabled">
                     <bool>false</bool>
                    </property>
                    <property name="calendarPopup">
                     <bool>true</bool>
                    </property>
                   </widget>
                  </item>
                  <item>
                   <widget class="QLabel" name="recallRunLabel">
                    <property name="text">
                     <string>Run:</string>
                    </property>
                   </widget>
                  </item>
                  <item>
                   <widget class="QSpinBox" name="recallRunBox">
                    <property name="enabled">
                     <bool>false</bool>
                    </property>
                    <property name="minimum">
                     <number>1</number>
                    </property>
                   </widget>
                  </item>
                 </layout>
                </item>
                <item>
                 <layout class="QHBoxLayout" name="horizontalLayout_5">
                  <item>
                   <widget class="QLabel" name="delayLabel">
                    <property name="text">
                     <string>Processing delay (s):</string>
                    </property>
                   </widget>
                  </item>
                  <item>
                   <widget class="QSpinBox" name="delayBox">
                    <property name="minimumSize">
                     <size>
                      <width>100</width>
                      <height>0</height>
                     </size>
                    </property>
                    <property name="maximumSize">
                     <size>
                      <width>100</width>
                      <height>16777215</height>
                     </size>
                    </property>
                   </widget>
                  </item>
                 </layout>
                </item>
                <item>
                 <layout class="QVBoxLayout" name="singleFileLayout">
                  <item>
                   <widget class="QCheckBox" name="loadTofCheck">
                    <property name="enabled">
                     <bool>false</bool>
                    </property>
                    <property name="minimumSize">
                     <size>
                      <width>100</width>
                      <height>0</height>
                     </size>
                    </property>
                    <property name="maximumSize">
                     <size>
                      <width>100</width>
                      <height>16777215</height>
                     </size>
                    </property>
                    <property name="text">
                     <string>Single File</string>
                    </property>
                   </widget>
                  </item>
                  <item>
                   <layout class="QHBoxLayout" name="fileFinderLayout">
                    <item>
                     <widget class="QLineEdit" name="singleFileLine">
                      <property name="enabled">
                       <bool>false</bool>
                      </property>
                      <property name="minimumSize">
                       <size>
                        <width>200</width>
                        <height>0</height>
                       </size>
                      </property>
                      <property name="maximumSize">
                       <size>
                        <width>200</width>
                        <height>16777215</height>
                       </size>
                      </property>
                     </widget>
                    </item>
                    <item>
                     <widget class="QPushButton" name="singleFileButton">
                      <property name="enabled">
                       <bool>false</bool>
                      </property>
                      <property name="minimumSize">
                       <size>
                        <width>25</width>
                        <height>0</height>
                       </size>
                      </property>
                      <property name="maximumSize">
                       <size>
                        <width>25</width>
                        <height>16777215</height>
                       </size>
                      </property>
                      <property name="text">
                       <string>...</string>
                      </property>
                     </widget>
                    </item>
                   </layout>
                  </item>
                 </layout>
                </item>
               </layout>
              </widget>
             </item>
             <item>
              <layout class="QHBoxLayout" name="camRunLayout">
               <item>
                <widget class="QPushButton" name="camRunButton">
                 <property name="maximumSize">
                  <size>
                   <width>100</width>
                   <height>16777215</height>
                  </size>
                 </property>
                 <property name="text">
                  <string>Run</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QComboBox" name="camModeCombo">
                 <property name="maximumSize">
                  <size>
                   <width>100</width>
                   <height>16777215</height>
                  </size>
                 </property>
                 <item>
                  <property name="text">
                   <string>Live</string>
                  </property>
                 </item>
                 <item>
                  <property name="text">
                   <string>Recall</string>
                  </property>
                 </item>
                </widget>
               </item>
               <item>
                <widget class="QPushButton" name="camUpdateButton">
                 <property name="maximumSize">
                  <size>
                   <width>100</width>
                   <height>16777215</height>
                  </size>
                 </property>
                 <property name="text">
                  <string>Update</string>
                 </property>
                </widget>
               </item>
              </layout>
             </item>
            </layout>
           </item>
           <item>
            <layout class="QVBoxLayout" name="camLayout">
             <item>
              <widget class="MplCanvasCam" name="camWidget" native="true"/>
             </item>
            </layout>
           </item>
          </layout>
         </widget>
         <widget class="QWidget" name="fitTab">
          <attribute name="title">
           <string>Fits</string>
          </attribute>
          <layout class="QHBoxLayout" name="horizontalLayout_4">
           <item>
            <layout class="QVBoxLayout" name="verticalLayout">
             <item>
              <widget class="QLabel" name="fitLabel">
               <property name="text">
                <string>Fit Results</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QTextEdit" name="fitText">
               <property name="minimumSize">
                <size>
                 <width>250</width>
                 <height>0</height>
                </size>
               </property>
               <property name="readOnly">
                <bool>true</bool>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item>
            <layout class="QVBoxLayout" name="analysisLayout">
             <item>
              <widget class="MplCanvasAnalysis" name="analysisWidget" native="true"/>
             </item>
            </layout>
           </item>
          </layout>
         </widget>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionCamera">
   <property name="text">
    <string>Camera</string>
   </property>
  </action>
  <action name="actionAbout">
   <property name="text">
    <string>About</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>MplCanvasCam</class>
   <extends>QWidget</extends>
   <header>app</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>MplCanvasAnalysis</class>
   <extends>QWidget</extends>
   <header>app</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
